lib/vsc/utils/generaloption.py

```diff
@@ -545,7 +554,7 @@ def is_value_a_commandline_option(self, opt, value, index=None):
             return "Value '%s' starts with a '-'" % value
 
         if not self.ALLOW_TYPO_AS_VALUE:
-            possibilities = self._long_opt.keys() + self._short_opt.keys()
+            possibilities = list(self._long_opt.keys()) + list(self._short_opt.keys())
```

Comment:
you can't add the `.keys` before converting/forcing to a list?

Reply:
no, because `.keys()` returns an iterator in Python 3, and you can't add iterators...